oak-blob-cloud-azure/src/main/java/org/apache/jackrabbit/oak/blob/cloud/azure/blobstorage/AzureBlobStoreBackend.java

```diff
@@ -324,46 +299,28 @@ public void write(DataIdentifier identifier, File file) throws DataStoreExceptio
 
             long len = file.length();
             LOG.debug("Blob write started. identifier={} length={}", key, len);
-            CloudBlockBlob blob = getAzureContainer().getBlockBlobReference(key);
+            BlockBlobClient blob = getAzureContainer().getBlobClient(key).getBlockBlobClient();
             if (!blob.exists()) {
                 addLastModified(blob);
-
-                BlobRequestOptions options = new BlobRequestOptions();
-                options.setConcurrentRequestCount(concurrentRequestCount);
```

Comment:
you removed this and didn't replaced in azure sdk 12. you can use `ParallelTransferOptions` and `BlockBlobClient.uploadWithResponse`